chia/types/blockchain_format/vdf.py

```diff
@@ -2,21 +2,22 @@
 
 import logging
 import traceback
-from dataclasses import dataclass
 from enum import IntEnum
 from functools import lru_cache
 from typing import Optional
 
+from chia_rs import VDFInfo, VDFProof
 from chiavdf import create_discriminant, verify_n_wesolowski
 
 from chia.consensus.constants import ConsensusConstants
 from chia.types.blockchain_format.classgroup import ClassgroupElement
 from chia.types.blockchain_format.sized_bytes import bytes32, bytes100
 from chia.util.ints import uint8, uint64
-from chia.util.streamable import Streamable, streamable
 
 log = logging.getLogger(__name__)
 
+__all__ = ["VDFInfo", "VDFProof"]
```

Comment:
Will we separately update the imports to get these from `chia_rs` directly?

Reply:
maybe down the line. I'm trying to make minimal updates to have all types be exported from the same place, to retain some backwards compatibility